manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WorldOfWarships.ModStation
+PackageVersion: 2.1.0.2
+Installers:
+- Architecture: x86

```

It looks like this program installs to `C:\Windows\Program Files` and not `C:\Windows\Program Files (x86)`
```suggestion
- Architecture: neutral
```